network-api/networkapi/mozfest/customblocks/card_block.py

```diff
@@ -0,0 +1,35 @@
+from wagtail import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+"""
+The following card and grid definitions are specifically for Mozfest.
+They are based on wagtailpages.pagemodels.customblocks.card_grid.py
+but have been modified to fit the Mozfest design.
+"""
+
+
+class MozfestCardGrid(blocks.StructBlock):
+    image = ImageChooserBlock()
+
+    alt_text = blocks.CharBlock(required=False, help_text="Alt text for card's image.")
+
+    title = blocks.CharBlock(help_text="Heading for the card.")
+    category = blocks.CharBlock(help_text="Category text for the card.", required=False)
+    date_meta = blocks.CharBlock(help_text="Date and time or other information.", required=False)
```

Comment:
I've called this date meta so it can support dates or other text as it might not always make sense for it to just be a date.. I am battling with both 're-use' requirements and 'too many blocks' requirements. :shrug: